ui/prometheus-plugin/src/plugins/prometheus-time-series-query/get-time-series-data.ts

```diff
@@ -64,6 +65,19 @@ export const getTimeSeriesData: TimeSeriesQueryPlugin<PrometheusTimeSeriesQueryS
   // TODO: What about error responses from Prom that have a response body?
   const result = response.data?.result ?? [];
 
+  // Custom display for response header warnings, configurable error responses display coming next
```

Comment:
Is this another bit of code that could be tested?

Reply:
Yeah definitely, I'm planning to add tests internally that will be helpful and when Eunice's DataQueryProvider updates are in, I'm going to come back to this, build out the warnings tags that will show in a Panel's header, and add tests with Playwright / Happo